Testing/CASLTests/Devices/Exceptions/AudioDeviceDoesNotExistExceptionTests.cs

```diff
@@ -17,45 +18,59 @@ public class AudioDeviceDoesNotExistExceptionTests
     [Fact]
     public void Ctor_WhenInvokedWithNoParam_CorrectlySetsMessage()
     {
+        // Arrange
+        var expected = "The audio device does not exist.";
+
         // Act
         var exception = new AudioDeviceDoesNotExistException();
 
         // Assert
-        Assert.Equal("The audio device does not exist.", exception.Message);
+        exception.Message.Should().Be(expected);
     }
 
     [Fact]
     public void Ctor_WhenInvokedWithSingleMessageParam_CorrectlySetsMessage()
     {
+        // Arrange
+        var expected  = "test-message";
+
         // Act
-        var exception = new AudioDeviceDoesNotExistException("test-message");
+        var exception = new AudioDeviceDoesNotExistException(expected);
 
         // Assert
-        Assert.Equal("test-message", exception.Message);
+        exception.Message.Should().Be(expected);
     }
 
     [Fact]
     public void Ctor_WhenInvokedWithMessageAndDeviceNameParams_CorrectlySetsMessage()
     {
+        // Arrange
+        var expectedDeviceName = "device";
+        var expectedMessage = "test-message";
+        var expectedExceptionMessage = $"DeviceName: {expectedDeviceName}\n{expectedMessage}";
+
```

Comment:
Please fix the unit test. 
I have a failing unit test here related to the exception message.
Refer to the suggestion below to see what it should be.

```suggestion
        // Arrange
        var expectedDeviceName = "device";
        var expectedMessage = "test-message";
        var expectedExceptionMessage = $"Device Name: {expectedDeviceName}\n{expectedMessage}";

```

Reply:
The suggestion has been applied!